.github/workflows/regression-report.yml

```diff
@@ -56,11 +63,22 @@ jobs:
               content: 'rocket'
             })
 
-      - run: 'echo We print it here for this action to work'
-        if: 'true'
-
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - name: Check PR status
+        id: check_pr_status
```

Comment:
Why you need this code 
If you have:
```
pull_request:
    types:
      - opened
```

Reply:
> Why you need this code If you have:
> 
> ```
> pull_request:
>     types:
>       - opened
> ```

<img width="706" alt="image" src="https://github.com/user-attachments/assets/9cdeb973-1cff-41d7-a92a-d3d5d1dc225a">

so, if the PR is inactive, it will mention this in the comments of the PR. Hence, i added this for conveniece. Otherwise, if i don't i saw that parse_comment section would pass. And, the rest two sections won't do anything i.e. make_report and send_message. i guess a new contributor would take time to understand why the worklfow ain't working.

Reply:
Let's not be so fancy. 
I need this preventative check only to avoid ddos attack on us on through active pull.
It should not even trigger action at all, look at this as performance optimization.

Please remove this block.

Reply:
> Let's not be so fancy. I need this preventative check only to avoid ddos attack on us on through active pull. It should not even trigger action at all, look at this as performance optimization.
> 
> Please remove this block.

done